build/configs/sabre-6quad/hello_smp/Make.defs

```diff
@@ -0,0 +1,163 @@
+###########################################################################
```

Comment:
1. We don't keep `Make.defs` separately. We use just one Make.defs which is located at build/configs/<board>.
Could you merge this there?
2. How about RTL8730E? This PR does not include that file change.

Reply:
1, Done
2. This PR is only for merging smp code on v7a and verifiying on sabrelite qemu. We will make a separate PR for rtl8730e after RTK provides us the smp code.